setup.cfg

```diff
@@ -71,7 +71,7 @@ dev =
     flake8==6.0.0
     hypothesis==6.75.2
     ipdb==0.13.13
-    ipython==8.13.2
+    ipython==8.12.0
```

Comment:
ipdb and ipython doesn't seem to be used, could we remove them? Didn't check all the packages installed